modules/postgres/postgres.go

```diff
@@ -295,8 +295,10 @@ func (c *PostgresContainer) Restore(ctx context.Context, opts ...SnapshotOption)
 
 	// execute the commands to restore the snapshot, in order
 	return c.execCommandsSQL(ctx,
-		// Drop the entire database by connecting to the postgres global database
-		fmt.Sprintf(`DROP DATABASE "%s" with (FORCE)`, c.dbName),
+		// Terminate all connections to the template database
+		fmt.Sprintf(`SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname = '%s' AND pid <> pg_backend_pid()`, snapshotName),
```

Comment:
question: Based on [the docs](https://www.postgresql.org/docs/current/sql-dropdatabase.html) FORCE already does the connection termination?

Reply:
Done